src/views/DoctorOrders/OrderCard/OrderCard.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import './OrderCard.css';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+
+function createData(
+  patientName: string,
+  patientDOB: string,
+  doctorName: string,
+  doctorContact: string,
+  doctorID: number,
+  doctorEmail: string,
+  drugNames: string,
+  drugPrice: number,
+  quanitities: number,
+  total: number,
+  pickupDate: string,
+  caseNumber: string,
+  dispenseStatus: string,
+) {
+  return { patientName, patientDOB, doctorName, doctorContact, doctorID, doctorEmail, drugNames, drugPrice, quanitities, total, pickupDate, caseNumber, dispenseStatus };
+}
+
+const rows = [
+  createData( "John Snow", '10/22/1980', 'Dr. Jane Betty Doe', '716-873-1557', 1122334455, 'jane.betty@myhospital.com', 'Isotretinoin 200 MG Oral Capsule', 200, 90, 18000, 'Tue Dec 13 2022', '43f575fe0ae14558976e4a71b7483242', 'Pending'),
+
+];
+
+export default function OrderCard() {
+  return (
+    <Card sx={{ minWidth: 275, mb: '10px' }}>
+      <CardContent>
+        {rows.map((row) => (
+          <Box>
+            <Typography variant="h5" component="div">
+              {row.patientName}
+            </Typography>
+            <Typography color="text.secondary">
+              DOB: {row.patientDOB}
+            </Typography>
+            <Typography sx={{ fontSize: 14 }} color="text.secondary" gutterBottom>
+              Case # {row.caseNumber}
+            </Typography>
+            <Typography sx={{ mb: 2 }} variant="h6">
+              {row.drugNames}
+            </Typography>
+          </Box>
+        ))}
+        <TableContainer component={Paper}>
```

Comment:
I don't know how much we care about design but the single-row table inside the elements which are organized into a list already strikes me as odd, though it mimics with the "original" design almost exactly in formatting the information, so I see where it's coming from.

I don't think this is something you need to change but I'd like to see the information organized in the space of each element in the list more compactly and with less room for the headers if we have extra time to make things look nice.

Reply:
I agree, originally was copying the style of the old pharmacy, but would like to explore other options for how to display it so it presents a bit nicer